obs_maven/repo.py

```diff
@@ -69,9 +82,9 @@ def parse_primary(self):
                         input_source.setByteStream(gzip_fd)
                         parser.parse(input_source)
                         self._rpms = handler.rpms.values()
-            except:
+                break
+            except OSError:
```

Comment:
I am only 90% sure this catches everything we need. The one we hit most often at the moment, `ConnectionResetError`, is a subclass of `OSError` and therefore handled.